.github/labeler.json

```
@@ -15,6 +15,11 @@
       { "type": "user", "pattern": "kyliau" },
       { "type": "user", "pattern": "spanicker" }
     ],
+    "ci-approved": [
```

```suggestion
    "CI approved": [
```
I think needs to be formatted the same with the space